internal/schema/schema.go

```diff
@@ -511,7 +511,13 @@ func parseDirectiveDef(l *common.Lexer) *types.DirectiveDefinition {
 		l.ConsumeToken(')')
 	}
 
-	l.ConsumeKeyword("on")
+	switch x := l.ConsumeIdent(); x {
+	case "on":
+	case "repeatable":
+		l.ConsumeKeyword("on")
```

Comment:
How does the above work if in your example you have two words `repeatable on`:
```
directive @repeatabledirective repeatable on SCALAR
```
 with one space between them and in the code above you consume one indent, then any of the words `on` or `repeatable`. Does this mean that you would consume the keyword `"on"` twice?

Reply:
Ah, yeah -- the `case "on"` is a no-op since Go doesn't fallthrough-by-default, but this is really easy to misread. I've tried to fix this by adding a comment (and tests to verify that `on on` (and `repeatable repeatable` for good measure) doesn't parse.)